RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate AddChannelTeamView to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce


- Create a `Team`, then click on `#` icon, and click on "edit/create" icon on the header, and the new header title should be `Add Channel to Team`, so this is the view migrated.
- Also, there is must be 2 buttons, one for `Create New` and another `Add Existing`

## Screenshots

## Types of changes




## Checklist



## Further comments
